react/wbooks/src/app/components/SignUp/index.js

```diff
@@ -34,7 +29,7 @@ class SignUp extends Component {
     return (
       <div className={`${styles.container} column background-wild-sand`}>
         <img src={imagePlaceholder} alt="Wolox logo" className={styles.woloxLogoImage} />
-        <form onSubmit={this.handleSignUp} className={`${styles.signUpForm} m-bottom-3`}>
+        <form className={`${styles.signUpForm} m-bottom-3`}>
```

Comment:
why are you removing the `onSubmit`? its better to submit a form rather than handling in a onClick function in a button